071-GitHubCopilot/Coach/Solution-03.md

```diff
@@ -0,0 +1,13 @@
+# Challenge 03 - Leveraging GitHub Copilot in Your Dev Workflow - Coach's Guide 
+
+[< Previous Solution](./Solution-02.md) - **[Home](./README.md)** - [Next Solution >](./Solution-04.md)
+
+## Notes & Guidance
+
+- This challenge is to show that Copilot can do more than just help with developers writing code.  Here we are going to also leverage Copilot to help with commit messages and a GitHub Action pipeline (yaml).  At the end of this challenge you can also discuss some other areas where Copilot may be able to help.  Copilot chat will likely be able to help scaffold a good portion of this.  Below are some examples:
+  - Helping with XML such as in APIM policies
+  - Infrastructure as code such as ARM/Bicep
+  - SQL Queries
+- To generate a commit message, they can go to VSCode, and in the source control section there will be these stars/diamonds next to the commit message box.  Selecting that will allow you to leverage Copilot to generate a commit message.  They can generate this after creating the pipeline to have something to checkin to.
+- This could also be a great time to talk about some of GitHub Copilot Enterprises other features such as PR message creation which is along the same concept. However this is a separate license and not part of the hack.
```

Comment:
should you be more specific about the license required?

Reply:
Added additional context for licenses and the different types